configs/tasks/rearrange/rearrange_easy_multi_agent.yaml

```diff
@@ -0,0 +1,169 @@
+GYM:
+    AUTO_NAME: RearrangeEasy
+    OBS_KEYS:
+    - robot_head_depth
+    - relative_resting_position
+    - obj_start_sensor
+    - obj_goal_sensor
+    - obj_start_gps_compass
+    - obj_goal_gps_compass
+    - joint
+    - is_holding
+ENVIRONMENT:
+    MAX_EPISODE_STEPS: 1500
+DATASET:
+    TYPE: RearrangeDataset-v0
+    SPLIT: train
+    DATA_PATH: data/datasets/replica_cad/rearrange/v1/{split}/rearrange_easy.json.gz
+    SCENES_DIR: data/replica_cad/
+TASK:
+    TYPE: RearrangeCompositeTask-v0
+    REWARD_MEASURE: move_obj_reward
+    SUCCESS_MEASURE: composite_success
+    SUCCESS_REWARD: 100.0
+    SLACK_REWARD: -0.01
+    END_ON_SUCCESS: true
+    COUNT_OBJ_COLLISIONS: true
+    COUNT_ROBOT_OBJ_COLLS: false
+    CONSTRAINT_VIOLATION_ENDS_EPISODE: false
+    CONSTRAINT_VIOLATION_DROPS_OBJECT: True
+    BASE_ANGLE_NOISE: 0.15
+    BASE_NOISE: 0.05
+    EASY_INIT: false
+    TASK_SPEC_BASE_PATH: configs/tasks/rearrange/pddl/
+    TASK_SPEC: rearrange_easy
+    PDDL_DOMAIN_DEF: replica_cad
+    OBJ_SUCC_THRESH: 0.3
+    ART_SUCC_THRESH: 0.15
+    SENSORS:
```

Comment:
It looks like it is possible to have heterogenous action space for the two agents but not heterogenous observation spaces. Can this be documented somewhere so users know the capabilities of the code?

Reply:
As @mathfac proposed: Let's document that we do not *yet* support heterogenous observation spaces for the robots and that the feature will come later. This way we can merge this code in for the release and wok on heterogenous observation spaces at a later date.

Reply:
What's the difficulty in adding it? Is'nt it just constructing the sensor suite twice (once for each agent) instead of duplicating? Maybe I am misunderstanding something here.

Reply:
Yeah @Skylion007 is correct that there's no real difficulty and this is already supported. The problem was the amount of repeated sensors would be huge. For that reason I added the auto sensor duplication code if the sensor is marked with `UsesRobotInterface`.

Reply:
Ah, I see. This is a robot specific issue.